js/analyses/getSubheadings.js

```diff
@@ -0,0 +1,25 @@
+/* @module analyses/getSubheadings */
```

Comment:
`getSubheadings` implies that we are returning the subheadings, but we are not we are matching keywords in subheadings. A better name would be `matchKeywordInSubheadings` or `countKeywordInSubheadings`